.changeset/silent-icons-deny.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/backend-app-api': patch
+---
+
+Index features on id instead of object.
+
+This will allow features added from an external package location to be loaded correctly,
+without the requirement of making backstage packages singletons.
```

Comment:
```suggestion
Extension points are now tracked via their ID rather than reference, in order to support package duplication.
```

feature -> extension, and figured there are quite a couple of situations where one might run into issues, but it all comes down to duplication of packages.